pkg/grpc/metrics/grpc_metrics_impl.go

```diff
@@ -4,9 +4,9 @@ import (
 	"context"
 	"runtime/debug"
 
-	lru "github.com/hashicorp/golang-lru/v2"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/lru"
```

Comment:
I would suggest minimizing the usage of our `lru` package. It's better to use Hashicorp LRU unless we have to use our own. And that should be only in the case of ExpriableLRU.

This applies to all 4 files where we did such changes.